wire-ios-system/Source/UserDefaults+Temporary.swift

```diff
@@ -55,8 +55,8 @@ private final class SuiteCleanUp {
         // ~/Library/Developer/CoreSimulator/Devices/<device id>/data/Containers/Data/Application/<app id>/Library/Preferences/<suiteName>.plist
         do {
             let fileManager = FileManager.default
-            let url = try fileManager
```

Comment:
@johnxnguyen @caldrian I'm pretty sure this is find but any concerns about this case?